crates/consensus/src/transaction/legacy.rs

```diff
@@ -336,4 +341,26 @@ mod tests {
         assert_eq!(*signed_tx.hash(), hash, "Expected same hash");
         assert_eq!(signed_tx.recover_signer().unwrap(), signer, "Recovering signer should pass.");
     }
+
+    #[test]
+    #[cfg(feature = "k256")]
+    fn decode_legacy_and_recover_signer() {
+        use crate::TxLegacy;
```

Comment:
this should include the etherscan url

Reply:
Will add the alloy issue which contains all the details